modelon/impact/client/entities/model_executable.py

```diff
@@ -253,7 +253,12 @@ def input(self) -> ModelExecutableInput:
         return ModelExecutableInput(self._get_info()["input"])
 
     def get_class_name(self) -> str:
-        """Return the model class name."""
+        """Deprecated, use 'get_model_name' attribute."""
+        logger.warning("This attribute is deprectated, use 'get_model_name' instead!")
```

Comment:
How about :
```
import warnings
@warnings.deprecated("The 'get_class_name' is deprecated, use 'get_model_name'")
def get_class_name(self) -> str:
```
if you think this is too much then please fix the spelling of deprec**t**ated